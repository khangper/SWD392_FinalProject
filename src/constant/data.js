export const API_KEY = "API_KEY";

export const FETCH_HOME_LIVESTREAM_REQUEST = "FETCH_HOME_LIVESTREAM_REQUEST";
export const FETCH_HOME_LIVESTREAM_SUCCESS = "FETCH_HOME_LIVESTREAM_SUCCESS";
export const FETCH_HOME_LIVESTREAM_FAILURE = "FETCH_HOME_LIVESTREAM_FAILURE";

export const FETCH_HOME_FEATURED_COURSES_REQUEST =
  "FETCH_HOME_FEATURED_COURSES_REQUEST";
export const FETCH_HOME_FEATURED_COURSES_SUCCESS =
  "FETCH_HOME_FEATURED_COURSES_SUCCESS";
export const FETCH_HOME_FEATURED_COURSES_FAILURE =
  "FETCH_HOME_FEATURED_COURSES_FAILURE";
// All Live Stream
export const FETCH_STREAMS_REQUEST = "FETCH_STREAMS_REQUEST";
export const FETCH_STREAMS_SUCCESS = "FETCH_STREAMS_SUCCESS";
export const FETCH_STREAMS_FAILURE = "FETCH_STREAMS_FAILURE";

//All Instructor
export const FETCH_INSTRUCTORS_REQUEST = "FETCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTORS_SUCCESS = "FETCH_INSTRUCTORS_SUCCESS";
export const FETCH_INSTRUCTORS_FAILURE = "FETCH_INSTRUCTORS_FAILURE";
export const SEARCH_INSTRUCTORS_REQUEST = "SEARCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTOR_BY_ID_REQUEST = "FETCH_INSTRUCTOR_BY_ID_REQUEST";

// SaveCourse.js
export const FETCH_SAVECOURSES_REQUEST = "FETCH_SAVECOURSES_REQUEST";
export const FETCH_SAVECOURSES_SUCCESS = "FETCH_SAVECOURSES_SUCCESS";
export const FETCH_SAVECOURSES_FAILURE = "FETCH_SAVECOURSES_FAILURE";
export const DELETE_SAVECOURSE_REQUEST = "DELETE_SAVECOURSE_REQUEST";
export const DELETE_SAVECOURSE_SUCCESS = "DELETE_SAVECOURSE_SUCCESS";
export const DELETE_SAVECOURSE_FAILURE = "DELETE_SAVECOURSE_FAILURE";

// Shoppingcart.js
export const ShoppingCart_API_URL =
  "https://66908916c0a7969efd9c67ed.mockapi.io/ojt-repo/ShoppingCart";
export const ADD_TO_CART = "ADD_TO_CART";
export const REMOVE_FROM_CART = "REMOVE_FROM_CART";
export const APPLY_DISCOUNT = "APPLY_DISCOUNT";
export const ADD_TO_CART_FAILURE = " ADD_TO_CART_FAILURE";
export const ADD_TO_CART_SUCCESS = "APPLY_DISCOUNT";

//MyCourse_Course
export const FETCH_COURSES_REQUEST = "FETCH_COURSES_REQUEST";
export const FETCH_COURSES_SUCCESS = "FETCH_COURSES_SUCCESS";
export const FETCH_COURSES_FAILURE = "FETCH_COURSES_FAILURE";

export const EDIT_COURSE_REQUEST = "EDIT_COURSE_REQUEST";
export const EDIT_COURSE_SUCCESS = "EDIT_COURSE_SUCCESS";
export const EDIT_COURSE_FAILURE = "EDIT_COURSE_FAILURE";

export const DELETE_COURSE_REQUEST = "DELETE_COURSE_REQUEST";
export const DELETE_COURSE_SUCCESS = "DELETE_COURSE_SUCCESS";
export const DELETE_COURSE_FAILURE = "DELETE_COURSE_FAILURE";

//UpcommingCourse_Course
export const FETCH_UPCOMMINGCOURSES_REQUEST = "FETCH_UPCOMMINGCOURSES_REQUEST";
export const FETCH_UPCOMMINGCOURSES_SUCCESS = "FETCH_UPCOMMINGCOURSES_SUCCESS";
export const FETCH_UPCOMMINGCOURSES_FAILURE = "FETCH_UPCOMMINGCOURSES_FAILURE";

export const EDIT_UPCOMMINGCOURSE_REQUEST = "EDIT_UPCOMMINGCOURSE_REQUEST";
export const EDIT_UPCOMMINGCOURSE_SUCCESS = "EDIT_UPCOMMINGCOURSE_SUCCESS";
export const EDIT_UPCOMMINGCOURSE_FAILURE = "EDIT_UPCOMMINGCOURSE_FAILURE";

export const DELETE_UPCOMMINGCOURSE_REQUEST = "DELETE_UPCOMMINGCOURSE_REQUEST";
export const DELETE_UPCOMMINGCOURSE_SUCCESS = "DELETE_UPCOMMINGCOURSE_SUCCESS";
export const DELETE_UPCOMMINGCOURSE_FAILURE = "DELETE_UPCOMMINGCOURSE_FAILURE";
