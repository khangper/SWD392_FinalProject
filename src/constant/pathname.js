export const PATH_NAME = {
  HOME: "/home",
  EXPLORE: "/explore",
  LIVE_OUTPUT: "/live_output/:id",
  ADD_LIVE_STREAM: "/add_livestream",
  ALL_LIVE_STREAM: "/all_livestream",
  ALL_INSTRUCTOR_LIST: "/allinstructorlist/:id",
  HELP: "/help",
  REPORT_HISTORY: "/reporthistory",
  SEND_FEEDBACK: "/sendfeedback",
  OTHER_INSTRUCTOR_VIEW: "/other-instructor-view/:id",
  MY_INSTRUCTOR_PROFILE: "/my-instructor-profile",
  SAVE_COURSES: "/save-courses",
  SETTING: "/setting",
  MY_COURSE: "/courses",
  LOGIN: "/login",
  FORGOT_PASSWORD: "/forgot-password",
  SIGN_UP: "/sign-up",
  SIGN_UP_STEP: "/SignUpStep",
  THANKS: "/thanks",
  ERROR_404: "/error404",
  INVOICE: "/invoice",
  PRESS: "/press",
  SEARCH_RESULT: "/search_result",
  PAID_MEMBERSHIP: "/paidmembership",
  ABOUT_US: "/about-us",
  OUR_BLOG: "/our-blog",
  BLOG_SINGLE_VIEW: "/blog-single-view",
  CHECKOUT: "/checkout",
  SHOPPING_CART: "/shoppingcart",
  HELP_VIEW_1: "/helpview1",
  HELP_VIEW_2: "/helpview2",
  COMPANY_DETAILS: "/company-details",
  CAREER: "/career",
  APPLY_JOB: "/apply-job",
  CONTACT_US: "/contactus",
  TERMS_OF_USE: "/termsofuse",
  CERTIFICATION_FILL_FORM: "/certificationfillform",
  CERTIFICATION_TEST: "/certificates-test",
  TEST_RESULT: "/test-result",
  MESSAGE: "/messages",
  COURSES_DETAIL_VIEW: "/courses-detail-view/:id",
  PREVIEWS: "/reviews",
  COMING_SOON: "/coming-soon",
  INS_DASHBOARD: "/ins-dashboard",
  PURCHASE_COURSE: "/PurchaseCourse",
  NOTIFICATON: "/notifications",
  DOWLOADCOURSE: "/dowloadcourse",
  CERTIFICATION_CENTER: "/certificationcenter",
  SITEMAP: "/sitemap",
  EARNING: "/earning",
  PAYOUT: "/payout",
  VERTIFICATION: "/vertification",
  MYCERTIFICATIONS: "/mycertifications",
  CREDIT: "/credits",
  STATEMENTIS: "/statementis",
  STATEMENTST: "/statementst",
  ANALYTICS: "/analytics",
  CREATE_NEW_COURSE: "/create-course",
  STD_DASHBOARD: "/student-dashboard",
  NOTIFICATIONSTUDENT: "/notificationsstudent",
};
