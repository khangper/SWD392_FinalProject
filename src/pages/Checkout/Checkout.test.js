import React from "react";
import { render, screen, fireEvent } from "@testing-library/react";
import Checkout from "../Checkout/Checkout";
import "@testing-library/jest-dom/extend-expect";
import { renderWithProviders } from "../../constant/test-util";

describe("Checkout", () => {
  beforeAll(() => {
    jest.spyOn(console, "error").mockImplementation(() => {});
    jest.spyOn(console, "log").mockImplementation(() => {});
  });

  afterAll(() => {
    console.error.mockRestore();
    console.log.mockRestore();
  });

  // input First Name

  it("renders the Checkout component", async () => {
    const { container } = renderWithProviders(<Checkout />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });

  it("handles whitespace input", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("First Name");

    // Simulate user typing whitespace
    fireEvent.change(input, { target: { value: "   " } });
    expect(input.value).toBe("   ");
  });

  it("handles special character input", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("First Name");

    // Simulate user typing special characters
    fireEvent.change(input, { target: { value: "@#$%^&*()!" } });
    expect(input.value).toBe("@#$%^&*()!");
  });

  it("handles whitespace input in the Last Name field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Last Name");

    // Simulate user typing whitespace
    fireEvent.change(input, { target: { value: "   " } });
    expect(input.value).toBe("   ");
  });

  it("handles special character input in the Last Name field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Last Name");

    // Simulate user typing special characters
    fireEvent.change(input, { target: { value: "@#$%^&*()!" } });
    expect(input.value).toBe("@#$%^&*()!");
  });

  //Last Name

  it("handles whitespace input in the Last Name field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Last Name");

    // Simulate user typing whitespace
    fireEvent.change(input, { target: { value: "   " } });
    expect(input.value).toBe("   ");
  });

  it("handles special character input in the Last Name field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Last Name");

    // Simulate user typing special characters
    fireEvent.change(input, { target: { value: "@#$%^&*()!" } });
    expect(input.value).toBe("@#$%^&*()!");
  });

  it("handles normal text input in the Last Name field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Last Name");

    // Simulate user typing a normal last name
    fireEvent.change(input, { target: { value: "Doe" } });
    expect(input.value).toBe("Doe");
  });

  it("handles whitespace input in the Academy Name field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Academy Name");

    // Simulate user typing whitespace
    fireEvent.change(input, { target: { value: "   " } });
    expect(input.value).toBe("   ");
  });

  it("handles special character input in the Academy Name field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Academy Name");

    // Simulate user typing special characters
    fireEvent.change(input, { target: { value: "@#$%^&*()!" } });
    expect(input.value).toBe("@#$%^&*()!");
  });

  it("handles normal text input in the Academy Name field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Academy Name");

    // Simulate user typing a normal academy name
    fireEvent.change(input, { target: { value: "Tech Academy" } });
    expect(input.value).toBe("Tech Academy");
  });

  // Postalcode

  it("handles whitespace input in the Postal Code field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Postal Code");

    // Simulate user typing whitespace
    fireEvent.change(input, { target: { value: "   " } });
    expect(input.value).toBe("   ");
  });

  it("handles special character input in the Postal Code field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Postal Code");

    // Simulate user typing special characters
    fireEvent.change(input, { target: { value: "@#$%^&*()!" } });
    expect(input.value).toBe("@#$%^&*()!");
  });

  it("handles normal text input in the Postal Code field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Postal Code");

    // Simulate user typing a normal postal code
    fireEvent.change(input, { target: { value: "ABCDE" } });
    expect(input.value).toBe("ABCDE");
  });

  it("handles numeric input in the Postal Code field", async () => {
    renderWithProviders(<Checkout />);
    const input = screen.getByPlaceholderText("Postal Code");

    // Simulate user typing a numeric postal code
    fireEvent.change(input, { target: { value: "12345" } });
    expect(input.value).toBe("12345");
  });
});
