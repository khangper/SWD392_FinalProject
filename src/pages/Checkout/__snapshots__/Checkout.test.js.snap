// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkout renders the Checkout component 1`] = `
<div>
  <div
    class="CheckoutPage"
  >
    <header
      class="header1"
    >
      <a
        href="/home"
      >
        <button
          class="header-link1"
        >
          Back To Cursus
        </button>
      </a>
      <a
        href="/home"
      >
        <img
          alt=""
          class="header-group-icon"
          src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo.svg"
        />
      </a>
      <img
        alt=""
        class="header-avatar"
        src="profile-img.jpg"
      />
    </header>
    <div
      class="Checkoutfix-grid-container"
    >
      <div
        class="HelpView-title125"
      >
        <div
          class="HelpView-titleleft"
        >
          <div
            class="HelpView-ttl121"
          >
            <nav
              aria-label="breadcrumb"
            >
              <ol
                class="HelpView-breadcrumb"
              >
                <li
                  class="HelpView-breadcrumb-item"
                >
                  <a
                    href="/home"
                  >
                    Home
                  </a>
                </li>
                <li
                  aria-current="page"
                  class="HelpView-breadcrumb-item active"
                >
                  /Paidmember ship/ Checkout
                </li>
              </ol>
            </nav>
          </div>
        </div>
      </div>
      <div
        class="HelpView-title126"
      >
        <h2>
          Checkout
        </h2>
      </div>
    </div>
    <div
      class="Checkout-Container"
    >
      <div
        class="Billing-Details-GridContainer"
      >
        <div
          class="Billing-Details"
        >
          <div
            class="checkout_titie"
          >
            <h4>
              Billing Details
            </h4>
            <img
              alt="line"
              src="https://gambolthemes.net/html-items/cursus-new-demo/images/line.svg"
            />
          </div>
          <div
            class="panel-group"
          >
            <div
              class=" panel-default"
            >
              <div
                class="panel-heading-checkout"
              >
                <span
                  class="edit-title"
                >
                  Edit Address
                </span>
                <span
                  class="edit-icon"
                >
                  âœš
                </span>
              </div>
              <div
                class="panel-collapse "
              >
                <div
                  class="panel-body basic_form"
                >
                  <form>
                    <div
                      class="grid-container"
                    >
                      <div
                        class="Gridcheckout"
                      >
                        <div
                          class="ui search focus mt-30 lbel25"
                        >
                          <label>
                            First Name*
                          </label>
                          <div
                            class="ui left icon input swdh11 swdh19"
                          >
                            <input
                              class="prompt srch_explore"
                              name="firstName"
                              placeholder="First Name"
                              type="text"
                              value="Joginder"
                            />
                          </div>
                        </div>
                      </div>
                      <div
                        class="Gridcheckout"
                      >
                        <div
                          class="ui search focus mt-30 lbel25"
                        >
                          <label>
                            Last Name*
                          </label>
                          <div
                            class="ui left icon input swdh11 swdh19"
                          >
                            <input
                              class="prompt srch_explore"
                              name="lastName"
                              placeholder="Last Name"
                              type="text"
                              value="Singh"
                            />
                          </div>
                        </div>
                      </div>
                      <div
                        class="Gridcheckout-full"
                      >
                        <div
                          class="ui search focus mt-30 lbel25"
                        >
                          <label>
                            Academy Name*
                          </label>
                          <div
                            class="ui left icon input swdh11 swdh19"
                          >
                            <input
                              class="prompt srch_explore"
                              name="academyName"
                              placeholder="Academy Name"
                              type="text"
                              value="Gambolthemes"
                            />
                          </div>
                          <div
                            class="help-block"
                          >
                            If you want your invoices addressed to an academy, leave blank to use your full name.
                          </div>
                        </div>
                      </div>
                      <div
                        class="Gridcheckout-full"
                      >
                        <div
                          class="mt-30 lbel25"
                        >
                          <label>
                            Country*
                          </label>
                        </div>
                        <input
                          class="prompt srch_explore"
                          name="country"
                          placeholder="Country"
                          type="text"
                          value="India"
                        />
                      </div>
                      <div
                        class="Gridcheckout"
                      >
                        <div
                          class="ui search focus mt-30 lbel25"
                        >
                          <label>
                            Postal Code*
                          </label>
                          <div
                            class="ui left icon input swdh11 swdh19"
                          >
                            <input
                              class="prompt srch_explore"
                              name="postalCode"
                              placeholder="Postal Code"
                              type="text"
                              value="302012"
                            />
                          </div>
                        </div>
                      </div>
                      <div
                        class="Gridcheckout"
                      >
                        <div
                          class="ui search focus mt-30 lbel25"
                        >
                          <label>
                            City*
                          </label>
                          <div
                            class="ui left icon input swdh11 swdh19"
                          >
                            <input
                              class="prompt srch_explore"
                              name="city"
                              placeholder="City"
                              type="text"
                              value="Jaipur"
                            />
                          </div>
                        </div>
                      </div>
                      <div
                        class="Gridcheckout-full"
                      >
                        <div
                          class="ui search focus mt-30 lbel25"
                        >
                          <label>
                            Address Line 1*
                          </label>
                          <div
                            class="ui left icon input swdh11 swdh19"
                          >
                            <input
                              class="prompt srch_explore"
                              name="address1"
                              placeholder="Address 1"
                              type="text"
                              value="B-778, JMD, Sitapura, Near Genpact"
                            />
                          </div>
                        </div>
                      </div>
                      <div
                        class="Gridcheckout-full"
                      >
                        <div
                          class="ui search focus mt-30 lbel25"
                        >
                          <label>
                            Address Line 2
                          </label>
                          <div
                            class="ui left icon input swdh11 swdh19"
                          >
                            <input
                              class="prompt srch_explore"
                              name="address2"
                              placeholder="Address 2"
                              type="text"
                              value=""
                            />
                          </div>
                        </div>
                      </div>
                      <div
                        class="Gridcheckout-full"
                      >
                        <button
                          class="save_address_btn"
                          type="submit"
                        >
                          Save Changes
                        </button>
                      </div>
                    </div>
                  </form>
                </div>
              </div>
            </div>
          </div>
          <div
            class="address_text"
          >
            <br />
                Joginder Singh
            <br />
                #1234 Street No. 45, Ward No. 60, Phase 3,
            <br />
                Shahid Karnail Singh Nagar, Near Pakhowal
            <br />
                Road.
            <br />
                Ludhiana, Punjab, 141013
            <br />
                India
            <br />
              
            <br />
          </div>
        </div>
        <div
          class="Billing-Details"
        >
          <div
            class="Select-Payment-Method "
          >
            <div
              class="checkout_titie"
            >
              <h4>
                Select Payment Method
              </h4>
              <img
                src="https://gambolthemes.net/html-items/cursus-new-demo/images/line.svg"
              />
            </div>
            <div
              class="Select-custom-tabs-container"
            >
              <ul
                class="Select-custom-tabs-nav"
                role="tablist"
              >
                <li
                  class="Select-custom-tabs-item"
                >
                  <a
                    class="Select-custom-tabs-link active"
                    href="#"
                  >
                    <i
                      class="fa fa-credit-card"
                    />
                    <br />
                     Credit/Debit Card
                  </a>
                </li>
                <li
                  class="Select-custom-tabs-item"
                >
                  <a
                    class="Select-custom-tabs-link"
                    href="#"
                  >
                    <i
                      class="fa fa-university"
                    />
                    <br />
                     Bank Transfer
                  </a>
                </li>
                <li
                  class="Select-custom-tabs-item"
                >
                  <a
                    class="Select-custom-tabs-link"
                    href="#"
                  >
                    <i
                      class="fa fa-paypal"
                    />
                    <br />
                     Paypal
                  </a>
                </li>
              </ul>
              <div
                class="custom-tabs-content"
              >
                <div
                  class="Select-custom-tabs-pane active"
                  id="credit-method-tab"
                >
                  <form
                    class="basic_form"
                  >
                    <div
                      class="gg-grid-container"
                    >
                      <div
                        class="grid-item holder-card"
                      >
                        <div
                          class="ui search focus lbel25"
                        >
                          <label>
                            Holder Name
                          </label>
                          <div
                            class="ui left icon input swdh11 swdh19"
                          >
                            <input
                              class="prompt srch_explore"
                              id="id_holdername"
                              name="holder[name]"
                              placeholder="Enter Holder Name"
                              required=""
                              type="text"
                            />
                          </div>
                        </div>
                      </div>
                      <div
                        class="grid-item holder-card"
                      >
                        <div
                          class="ui search focus lbel25"
                        >
                          <label>
                            Card Number
                          </label>
                          <div
                            class="ui left icon input swdh11 swdh19"
                          >
                            <input
                              class="prompt srch_explore"
                              name="card[number]"
                              placeholder="Card #"
                              type="text"
                            />
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      class="Ggg-grid-container"
                    >
                      <div
                        class="grid-item expiration"
                      >
                        <div
                          class="ui search focus lbel25"
                        >
                          <label>
                            Expiration Month
                          </label>
                          <select
                            class="selectpicker"
                            data-size="5"
                            name="card[expire-month]"
                          >
                            <option
                              value=""
                            >
                              Month
                            </option>
                            <option
                              value="1"
                            >
                              January
                            </option>
                            <option
                              value="2"
                            >
                              February
                            </option>
                            <option
                              value="3"
                            >
                              March
                            </option>
                            <option
                              value="4"
                            >
                              April
                            </option>
                            <option
                              value="5"
                            >
                              May
                            </option>
                            <option
                              value="6"
                            >
                              June
                            </option>
                            <option
                              value="7"
                            >
                              July
                            </option>
                            <option
                              value="8"
                            >
                              August
                            </option>
                            <option
                              value="9"
                            >
                              September
                            </option>
                            <option
                              value="10"
                            >
                              October
                            </option>
                            <option
                              value="11"
                            >
                              November
                            </option>
                            <option
                              value="12"
                            >
                              December
                            </option>
                          </select>
                        </div>
                      </div>
                      <div
                        class="grid-item expiration"
                      >
                        <div
                          class="ui search focus lbel25"
                        >
                          <label>
                            Expiration Year
                          </label>
                          <div
                            class="ui left icon input swdh11 swdh19"
                          >
                            <input
                              class="prompt srch_explore"
                              name="card[expire-year]"
                              placeholder="Year"
                              type="text"
                            />
                          </div>
                        </div>
                      </div>
                      <div
                        class="grid-item expiration"
                      >
                        <div
                          class="ui search focus lbel25"
                        >
                          <label>
                            CVC
                          </label>
                          <div
                            class="ui left icon input swdh11 swdh19"
                          >
                            <input
                              class="prompt srch_explore"
                              name="card[cvc]"
                              placeholder="CVC"
                              type="text"
                            />
                          </div>
                        </div>
                      </div>
                    </div>
                  </form>
                </div>
                <div
                  class="Select-custom-tabs-pane"
                  id="bank-method-tab"
                >
                  <form
                    class="CheckoutCC-grid-container"
                  >
                    <div
                      class="item"
                    >
                      <div
                        class="ui search focus mt-30 lbel25"
                      >
                        <label>
                          Account Holder Name
                        </label>
                        <div
                          class="ui left icon input swdh11 swdh19"
                        >
                          <input
                            class="prompt srch_explore"
                            name="account[holdername]"
                            placeholder="Enter Your Full Name"
                            type="text"
                            value=""
                          />
                        </div>
                      </div>
                    </div>
                    <div
                      class="item"
                    >
                      <div
                        class="ui search focus mt-30 lbel25"
                      >
                        <label>
                          Account Number
                        </label>
                        <div
                          class="ui left icon input swdh11 swdh19"
                        >
                          <input
                            class="prompt srch_explore"
                            name="Account[number]"
                            placeholder="Enter Account Number"
                            type="text"
                          />
                        </div>
                      </div>
                    </div>
                    <div
                      class="item"
                    >
                      <div
                        class="lbel25 mt-30"
                      >
                        <label>
                          Bank Name
                        </label>
                        <select
                          class="ui hj145 dropdown cntry152 prompt srch_explore"
                          name="Bank[name]"
                        >
                          <option
                            value=""
                          >
                            State Bank of India
                          </option>
                          <option
                            value="1"
                          >
                            Indian Bank
                          </option>
                          <option
                            value="2"
                          >
                            ICICI Bank
                          </option>
                          <option
                            value="3"
                          >
                            HDFC Bank
                          </option>
                          <option
                            value="4"
                          >
                            Yes Bank
                          </option>
                          <option
                            value="5"
                          >
                            Oriental Bank
                          </option>
                          <option
                            value="6"
                          >
                            Punjab National Bank
                          </option>
                        </select>
                      </div>
                    </div>
                    <div
                      class="item"
                    >
                      <div
                        class="ui search focus mt-30 lbel25"
                      >
                        <label>
                          IFSC Code
                        </label>
                        <div
                          class="ui left icon input swdh11 swdh19"
                        >
                          <input
                            class="prompt srch_explore"
                            name="IFSC[code]"
                            placeholder="Enter IFSC Code"
                            type="text"
                          />
                        </div>
                      </div>
                    </div>
                  </form>
                </div>
                <div
                  class="Select-custom-tabs-pane"
                  id="paypal-method-tab"
                >
                  <div
                    class="Paypal-grid-container"
                  >
                    <div
                      class="item"
                    >
                      <p
                        class="t-body"
                      >
                        After payment via PayPal's secure checkout, we will send you a link to download your files.
                      </p>
                      <div
                        class="media h-mt2"
                      >
                        <div
                          class="media__item -align-center"
                        >
                          <p
                            class="t2-body h-m0"
                          >
                            PayPal accepts
                          </p>
                        </div>
                        <div
                          class="media__body"
                        >
                          <ul
                            class="financial-institutes"
                            id="paypal-gateway"
                          >
                            <li
                              class="financial-institutes__logo"
                            >
                              <img
                                alt="Visa"
                                src="https://gambolthemes.net/html-items/cursus-new-demo/images/membership/pyicon-1.svg"
                                title="Visa"
                              />
                            </li>
                            <li
                              class="financial-institutes__logo"
                            >
                              <img
                                alt="MasterCard"
                                src="https://gambolthemes.net/html-items/cursus-new-demo/images/membership/pyicon-2.svg"
                                title="MasterCard"
                              />
                            </li>
                            <li
                              class="financial-institutes__logo"
                            >
                              <img
                                alt="American Express"
                                src="https://gambolthemes.net/html-items/cursus-new-demo/images/membership/pyicon-3.svg"
                                title="American Express"
                              />
                            </li>
                            <li
                              class="financial-institutes__logo"
                            >
                              <img
                                alt="Discover"
                                src="https://gambolthemes.net/html-items/cursus-new-demo/images/membership/pyicon-4.svg"
                                title="Discover"
                              />
                            </li>
                            <li
                              class="financial-institutes__logo"
                            >
                              <img
                                alt="China UnionPay"
                                src="https://gambolthemes.net/html-items/cursus-new-demo/images/membership/pyicon-5.svg"
                                title="China UnionPay"
                              />
                            </li>
                          </ul>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="OrderDetail"
          >
            <div
              class="checkout_titie"
            >
              <h4>
                Order Detail
              </h4>
              <img
                alt="line"
                src="https://gambolthemes.net/html-items/cursus-new-demo/images/line.svg"
              />
            </div>
            <div
              class="order_dt_section"
            >
              <div
                class="order_title"
              >
                <h4>
                  Baby Plan
                </h4>
                <div
                  class="order_price"
                >
                  $
                  0.00
                </div>
              </div>
              <div
                class="order_title"
              >
                <h6>
                  Taxes(GST)
                </h6>
                <div
                  class="order_price"
                >
                  0.00
                </div>
              </div>
              <div
                class="order_title"
              >
                <h3>
                  Total
                </h3>
                <div
                  class="order_price"
                >
                  $
                  0.00
                </div>
              </div>
              <button
                class="chckot_btn"
                type="submit"
              >
                Confirm Checkout
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        class="OrderSumury-GridContainer"
      >
        <div
          class="OrderSumury"
        >
          <div
            class="checkout_titie"
          >
            <h4>
              OrderSumury
            </h4>
            <img
              src="https://gambolthemes.net/html-items/cursus-new-demo/images/line.svg"
            />
          </div>
          <div
            class="order_dt_section"
          >
            <div
              class="order_title"
            >
              <h4>
                Baby Plan
              </h4>
              <div
                class="order_price"
              >
                $
                0.00
              </div>
            </div>
            <div
              class="order_title"
            >
              <h6>
                Taxes(GST)
              </h6>
              <div
                class="order_price"
              >
                0.00
              </div>
            </div>
            <div
              class="order_title"
            >
              <h2>
                Total
              </h2>
              <div
                class="order_price"
              >
                $
                0.00
              </div>
            </div>
            <div
              class="scr_text"
            >
              Secure checkout
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="footer footer-large-container"
    >
      <div
        class="footer-links"
      >
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/about-us"
          >
            About
          </a>
          <a
            class="link"
            href="/blog-single-view/:id"
          >
            Blog
          </a>
          <a
            class="link"
            href="/career"
          >
            Careers
          </a>
          <a
            class="link"
            href="/press"
          >
            Press
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/help"
          >
            Help
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Advertise
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Developers
          </a>
          <a
            class="link"
            href="/contactus"
          >
            Contact Us
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/termsofuse"
          >
            Copyright Policy
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Terms
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Privacy Policy
          </a>
          <a
            class="link"
            href="/sitemap"
          >
            Sitemap
          </a>
        </div>
        <div
          class="footer-column"
        >
          <button
            class="teach-button"
          >
            Teach on Cursus
          </button>
          <div
            class="language-selector"
          >
            <button>
              Language 
              <span>
                â–¼
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="footer-bottom"
      >
        <div
          class="footer-copy"
        >
          <img
            alt=""
            src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
          />
          Â© 2024 
          <strong>
            Cursus
          </strong>
          . All Rights Reserved.
        </div>
        <div
          class="footer-social"
        >
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="facebook.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="x.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="google+.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="linkedin.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="insta.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="pinterest.png"
            />
          </a>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
